src/include/simeng/RegisterFileSet.hh

```diff
@@ -16,11 +16,30 @@ struct Register {
    * architectural register, depending on point of usage. */
   uint16_t tag;
 
+  /** Default Constructor. */
+  Register(){};
+
+  Register(uint8_t type, uint16_t tag) : type(type), tag(tag){};
```

Comment:
Small comment on this Constructor please and Doxygen style comment

Reply:
Added